keywords/LiteServiOS.py

```diff
@@ -137,6 +144,17 @@ def install(self):
                 time.sleep(1)
                 continue
 
+        # Get the device ID
+        list_output = subprocess.Popen(["xcrun", "simctl", "list"], stdout=subprocess.PIPE)
```

Comment:
I am using check_output in the next line to grep Booted. From my research, this is equivalent to bash's pipe feature.